recipes-devtools/firmware-package-tool/files/update.conf.json

```diff
@@ -0,0 +1,34 @@
+{
+  "firmware": [
+    {
+      "description": "IOT2050 PG1 Bootloader Release V01.01.01",
+      "name": "iot2050-pg1-image-boot.bin",
+      "version": "V01.01.01",
+      "target_boards": [
+        "SIMATIC IOT2050-BASIC",
+        "SIMATIC IOT2050 Basic",
+        "SIMATIC IOT2050-ADVANCED",
+        "SIMATIC IOT2050 Advanced"
+      ]
+    },
+    {
+      "description": "IOT2050 PG2 Bootloader Release V01.01.01",
+      "name": "iot2050-pg2-image-boot.bin",
+      "version": "V01.01.01",
+      "target_boards": [
+        "SIMATIC IOT2050 Basic PG2",
+        "SIMATIC IOT2050 Advanced PG2"
+      ]
+    }
+  ],
+  "target_os": [
+    {
+      "type": "Example Image",
+      "key": "BUILD_ID",
+      "min_version": "V01.01.01"
+    }
+  ],
+  "suggest_preserved_uboot_env": [
+    "boot_targets"
+  ]
+}
```

Comment:
Long-term speaking, this should be replaced by the CLI interface.

This json is more for an internal implementation details/choice, not an interface.

But I am ok with it for the 1st version of this packaging tool.